dashboard/app/models/programming_class.rb

```diff
@@ -110,7 +110,8 @@ def summarize_for_edit
       tips: tips || '',
       syntax: syntax || '',
       externalDocumentation: external_documentation || '',
-      categoryKey: programming_environment_category&.key || ''
+      categoryKey: programming_environment_category&.key || '',
+      showUrl: programming_environment_programming_class_path(programming_environment.name, key)
```

Comment:
(minor) This name is slightly confusing because I initially read 'show' as a verb, but I don't really have a better suggestion.

Reply:
Ah I see that. `show` is based on the rails route term, which is why I picked it